go/account/templates/account/account_summary_mail.txt

```diff
@@ -20,8 +20,8 @@ Totals per conversation type:
 {{type}}
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 {% for conversation in conversations %}{{conversation.name}}
-    Sent: {{conversation.count_sent_messages}} to {{conversation.count_outbound_uniques }} uniques.
-    Received: {{conversation.count_replies}} from {{conversation.count_inbound_uniques }} uniques.
+    Sent: {{conversation.count_outbound_messages}} to {{conversation.count_outbound_uniques }} uniques.
+    Received: {{conversation.count_inbound_messages}} from {{conversation.count_inbound_uniques }} uniques.
```

Comment:
Those are methods I renamed. We consistently use `inbound` and `outbound` in most places, but had `replies` and `sent_messages` on the conversation wrapper.